libs/react-components/src/tedi/components/buttons/floating-button/floating-button.module.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../../styles/mixins';
+@import '../../../../shared/styles/bootstrap-utility/breakpoints';
+@import '../button-content/button-content.module';
+
+.tedi-floating-button {
+  box-shadow: 0 4px 10px 0 rgb(0 0 0 / 14%);
+
+  &--horizontal {
+    @include responsive-styles(border-radius, button-radius-default);
+  }
+
+  &--vertical {
+    transform: rotate(-90deg);
+
+    @include responsive-styles(border-radius, button-radius-sm);
+  }
+
+  &.tedi-floating-button--primary {
+    @include button-variant(
+      var(--button-floating-primary-background-default),
+      var(--button-floating-primary-border-default),
+      var(--button-floating-primary-text-default),
+      var(--button-floating-primary-background-hover),
+      var(--button-floating-primary-border-hover),
+      var(--button-floating-primary-text-default),
+      var(--button-floating-primary-border-focus),
+      var(--button-floating-primary-background-focus),
+      var(--button-floating-primary-text-default),
+      var(--button-floating-primary-border-active),
+      var(--button-floating-primary-background-active)
+    );
+  }
+
+  &.tedi-floating-button--secondary {
+    @include button-variant(
+      var(--button-floating-secondary-background-default),
+      var(--button-floating-secondary-border-default),
+      var(--button-floating-secondary-text-default),
+      var(--button-floating-secondary-background-hover),
+      var(--button-floating-secondary-border-hover),
+      var(--button-floating-secondary-text-hover),
+      var(--button-floating-primary-border-focus),
+      var(--button-floating-secondary-background-focus),
+      var(--button-floating-secondary-text-active),
+      var(--button-floating-secondary-border-active),
+      var(--button-floating-secondary-background-active)
+    );
+  }
+
+  &--medium {
+    height: 44px;
```

Comment:
Maybe add these into variables at the top of the css file e.g
```
$btn-height: 2.75rem;
$btn-height-small: 2.5rem;
$btn-height-large: 3rem;
```
and then assign `height: $btn-height;`
its better to keep them in rems too